### ISSUE TITLE
Chapter 11

### ISSUE THREAD

**Issue by codingedgar** [purescript-contrib/purescript-book#412]
Hi, thank you so much for this project! it is really helping me understand purescript 💛

Chapter 11 has this code:

### typo

```diff
- `ordered-collections`, which provides data typs for immutable maps and sets
+ `ordered-collections`, which provides data types for immutable maps and sets 
```

at https://github.com/purescript-contrib/purescript-book/blob/18706f27b75da06c53b5f85328bf3e964692957b/text/chapter11.md?plain=1#L11

### example for the reader monad
While the Writer monad a very understandable example with a final `PSCi` demo, the reader does not, which is hard to follow, i I'm hoping that it explains itself later tho, but it would be nice to have some different demos (PSCi examples) of the reader monad.

**Comment by milesfrain** [comment]
We don't yet have a good PSCi example of using the reader monad.
If you'd like to create an example, I'd be happy to include it in the text. It could even be different than the current user/admin snippets.
